src/assets/TokenListController.test.ts

```diff
@@ -11,17 +11,29 @@ import {
   TokenListController,
   TokenListStateChange,
   GetTokenListState,
+  TokenListMap,
+  IconPath,
+  MediaExtType,
 } from './TokenListController';
 
 const name = 'TokenListController';
 const TOKEN_END_POINT_API = 'https://token-api.airswap-prod.codefi.network';
 const timestamp = Date.now();
 
-const staticTokenList: any = {};
-for (const tokenAddress in contractmap) {
-  const { erc20, logo, ...token } = contractmap[tokenAddress];
+const staticTokenList: TokenListMap = {};
+for (const tokenAddress in contractMap) {
+  const { erc20, logo: filePath, ...token } = contractMap[tokenAddress];
+  const extType = filePath.split('.')[1].toUpperCase() as MediaExtType;
```

Comment:
this isn't going to work with filenames with more than just the extension (i know we don't have that here) but `my.cool.token.jpg` would break here:

![image](https://user-images.githubusercontent.com/675259/127569835-18226512-3dd5-4b2b-aed8-4bd870736bc5.png)

additionally it's just slow to use `.split` when all you really want is a string anyway. I think using substring methods is about 80% faster. you probably want:
```
filePath.substr(filePath.lastIndexOf('.') + 1)
```